File: contests/0509/A/main.c
// https://codeforces.com/contest/509/problem/A

#include <stdio.h>

int main() {
	int n;
	int a[10][10];
	int i, j;

	scanf("%d", &n);

	for (i = 0; i < n; i++) {
		a[0][i] = 1;
	}

	for (i = 1; i < n; i++) {
		a[i][i] = a[i - 1][i] + a[i - 1][i];
		for (j = i + 1; j < n; j++) {
			a[i][j] = a[i - 1][j] + a[i][j - 1];
		}
	}

	printf("%d\n", a[n - 1][n - 1]);

	return 0;
}

File: contests/0509/A/main.py
# https://codeforces.com/contest/509/problem/A

n = int(input())

range_n = range(n)
range_1_n = range(1, n)

m = [[0 for _ in range_n] for _ in range_n]

for i in range_n:
	m[0][i] = m[i][0] = 1

for i in range_1_n:
	for j in range_1_n:
		m[i][j] = m[i - 1][j] + m[i][j - 1]

print(m[n - 1][n - 1])
